### ISSUE TITLE
Propagate release labels for release from helmfile to helm release labels

### ISSUE THREAD

**Issue by dm3ch** [roboll/helmfile#2044]
Sometimes it's useful to filter helm releases using kubernetes-like filtering based on labels.

Filtering of labels have been implemented some time ago in helm - https://github.com/helm/helm/pull/8532
Unfortunately, right now there's a lack of adding labels to release functionality in helm.
But there's a feature request for adding this functionality (https://github.com/helm/helm/issues/7007) and there're PRs implementing that (https://github.com/helm/helm/pull/8099 https://github.com/helm/helm/pull/9087 https://github.com/helm/helm/pull/10533).

I believe it would be extremely useful to support the propagation of labels defined in helmfile.yaml to helm release when one of PRs mentioned above would be merged.

P.S. I also believe that it could be useful for implementation of https://github.com/roboll/helmfile/issues/194

**Comment by dm3ch** [comment]
Finally, https://github.com/helm/helm/pull/10533 have been merged, so now It's possible to implement this feature

**Comment by yxxhero** [comment]
@dm3ch please post a new issue in helmfile/helmfile.

**Comment by dm3ch** [comment]
@yxxhero It's a feature request and helmfile/helmfile seems to use discussions to track feature requests. Should I create it as a issue or as a discussion thread?

**Comment by yxxhero** [comment]
@dm3ch no. helmfile/helmfile is the new repo of reboll/helmfile. you can post a new discussion.

**Comment by yxxhero** [comment]
see: https://github.com/helmfile/helmfile/pull/1046
